libgoal/libgoal.go

```diff
@@ -952,6 +978,8 @@ func (c *Client) AddParticipationKey(keyfile string) (resp generated.PostPartici
 	}
 
 	return algod.PostParticipationKey(data)
+
+	// PKI TODO: Install state proof keys here.
```

Comment:
Do we really have a TODO here?
If I understand it correctly, at the point the entire (old) participation db is sent over the REST API, so state proof keys are already included there

the todo would be relevant when we won't send the entire file